rpm.spec

```diff
@@ -27,18 +28,17 @@ on Linux bridges when MSTP is used.
 ./autogen.sh
 %define _exec_prefix %{nil}
 %define _libexecdir /lib
-%configure
-make %{?_smp_mflags}
+%configure --with-systemdunitdir=%{_unitdir}
+%make_build
 
 %install
-rm -rf %{buildroot}
-make install DESTDIR=%{buildroot}
+%make_install
 
-sed -i -e 's|/etc/network/interfaces|/etc/sysconfig/network-scripts/bridge-stp|g' %{buildroot}/%{_libexecdir}/mstpctl-utils/ifquery
-sed -i -e 's|/etc/network/interfaces|/etc/sysconfig/network-scripts/bridge-stp|g' %{buildroot}/%{_libexecdir}/mstpctl-utils/mstp_config_bridge
+sed -i -e 's|/etc/network/interfaces|%{_sysconfdir}/sysconfig/network-scripts/bridge-stp|g' %{buildroot}%{_libexecdir}/mstpctl-utils/ifquery
+sed -i -e 's|/etc/network/interfaces|%{_sysconfdir}/sysconfig/network-scripts/bridge-stp|g' %{buildroot}%{_libexecdir}/mstpctl-utils/mstp_config_bridge
 
-mkdir -p %{buildroot}/%{_sysconfdir}/sysconfig/network-scripts
-cat <<END > %{buildroot}/%{_sysconfdir}/sysconfig/network-scripts/bridge-stp
+mkdir -p %{buildroot}%{_sysconfdir}/sysconfig/network-scripts
+cat <<"END" > %{buildroot}%{_sysconfdir}/sysconfig/network-scripts/bridge-stp
```

Comment:
are you sure that `END` needs quoting?
i'm a little paranoid about that;

Reply:
Yes, "END" means the content of the here-doc isn't parsed for $VAR refs and "`" executables... the old version tried to run "man mstpctl-utils-interfaces" and placed the output (empty) in the resulting file.  "END" means all the content is included as is.